src/formpack/reporting/export.py

```diff
@@ -526,3 +528,83 @@ def __get_extra_mapping_values(self, section):
+        file containing one SPSS file per translation. This includes *no* data!
+
+        :param output_file: a file-like object opened for writing
+        '''
+        all_versions = self.formpack.versions.values()
+        all_translations = set()
+        map(all_translations.update, [v.translations for v in all_versions])
+        all_fields = self.formpack.get_fields_for_versions()
+
+        with zipfile.ZipFile(output_file, 'w', zipfile.ZIP_DEFLATED) as z_out:
+            for translation in all_translations:
+                '''
+                For each translation, we need to produce a dictionary like:
+                    {
+                        'question_name1': {
+                            'label': 'I am the label of question 1!',
+                            'values': {
+                                'option1': 'Label option 1',
+                                'option2': 'Label option 2'
+                            }
+                        },
+                        'question_name2': {
+                            'label': 'I am question 2, unconstrained by
+                                     'predetermined values!'
+                        }
+                    }
+                '''
+                question_dict = OrderedDict()
+                for field in all_fields:
+                    # Even with `multiple_select='summary'`, we can still get
+                    # multiple names and labels per question for things like
+                    # `FormGPSField` (`geopoint`)
+                    xml_names = field.get_labels(lang=UNSPECIFIED_TRANSLATION,
+                                                 multiple_select='summary')
+                    assert xml_names[0] == field.name
+                    labels = field.get_labels(lang=translation,
+                                              multiple_select='summary')
+                    for name, label in zip(xml_names, labels):
+                        question_dict[name] = {
+                            'label': label,
+                            'data_type': field.data_type,
+                        }
+                    if hasattr(field, 'choice'):
+                        choices = OrderedDict()
+                        for option in field.choice.options.keys():
+                            choices[option] = field.get_translation(
+                                val=option, lang=translation
+                            )
+                        question_dict[field.name]['values'] = choices
+                # Convert the question/choice names and labels into SPSS
+                # commands
+                spss_label_commands = spss_labels_from_variables_dict(
+                    question_dict)
+                # Write the SPSS commands into a file for this particular
+                # language
+                title = self.formpack.title
+                rest_of_filename = ' - '.join(
+                    ('', translation, 'SPSS labels.sps')
```

Comment:
We may want to assert `translation` is not None otherwise this line raises an Exception. 
Maybe we could raise the Exception at the beginning of the method for forms without translations.